config/config_params.go

```diff
@@ -240,12 +248,30 @@ func (config *Config) UpdateFrom(rawData map[string]string, source Source) (chan
 	return
 }
 
-func (c *Config) InterfacePrefixes() []string {
-	return strings.Split(c.InterfacePrefix, ",")
+func (config *Config) InterfacePrefixes() []string {
+	return strings.Split(config.InterfacePrefix, ",")
```

Comment:
Sorry about that. It popped up as a recommended change (by `gofmt`?) through my IDE, since all the other functions related to `Config` had `config` instead of `c`. 

I actually had a bunch of other changes like this one, but quickly realized this is not the right place for it.

Reply:
Fixed.